lib/libevent/libevent.nit

```diff
@@ -470,20 +510,33 @@ class ConnectionFactory
 	do
 		var base = listener.base
 		var bev = new NativeBufferEvent.socket(base, fd, bev_opt_close_on_free)
-
-		# Human representation of remote client address
-		var addr_len = 46 # Longest possible IPv6 address + null byte
-		var addr_buf = new CString(addr_len)
-		addr_buf = addrin_to_address(addrin, addr_buf, addr_len)
-		var addr = if addr_buf.address_is_null then
-				"Unknown address"
-			else addr_buf.to_s
-
+		var addr=""
+
+		if not listener isa UnixConnectionListener then
```

Comment:
You should move this logic to something like `ConnectionListene::remote_address` and subclasses, including one for UNIX sockets. This address is used in logs so event a general "UNIX socket" would be better than nothing.

Reply:
I don't understand. You want me to move `var addr` to `ConnectionListener::remote_address` ?

Reply:
Well, I was talking about the block of code inside this condition, so that `addr` get its value from a method call. This kind of type check to customize a behavior is usually better done by a method and polymorphism.